docs/securecore.butane

```diff
@@ -0,0 +1,42 @@
+variant: fcos
+version: 1.5.0
+passwd:
+  users:
+    - name: core
+      groups:
+        - wheel
+      password_hash: $y$j9T$mefBCJbp/a49aSkTT4hpE1$6BXtrIuV8856t4A9r/R1GW4aR9eKXxsmB8FXt56Hx70 # 'secureblue'
+      ssh_authorized_keys:
+        - ssh-ed25519 <key>
+storage:
+  files:
+    - path: /opt/install_secureblue.sh
+      contents:
+        source: https://raw.githubusercontent.com/secureblue/secureblue/refs/heads/live/files/system/usr/share/secureblue/install_secureblue.sh
+        verification:
+          hash: sha256-d4ba5bfb556e9d1e3789a02fab2ab2f871033cc6b1712945cdfb9ce4375eafe5
+      mode: 0755
+    - path: /var/home/core/.bash_profile
+      overwrite: false
+      append:
+      - inline: |
+          echo "[+] Disabling zincati"
+          sudo systemctl disable --now zincati.service 2>/dev/null
+          echo "[+] Stopping rpm-ostreed-automatic"
+          sudo systemctl stop rpm-ostreed-automatic.timer rpm-ostreed-automatic.service 2>/dev/null
```

Comment:
this should get reenabled before reboot i think

Reply:
only zincati is disabled as per prereqs and rpm-ostreed will be started like normal upon reboot